Bio/SearchIO/HmmerIO/hmmer3_domtab.py

```diff
@@ -169,14 +167,14 @@ class Hmmer3DomtabHmmhitIndexer(Hmmer3TabIndexer):
     coordinates are hit coordinates."""
 
     _parser = Hmmer3DomtabHmmhitParser
-    _query_id_idx = 3
+    _qury_id_idx = 3
```

Comment:
These are so strange, as I didn't have to touch the code beyond the blank lines.